spaces/diadata/index.json

```
@@ -0,0 +1,27 @@
+{
+  "token": "0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419",
+  "name": "DIAData",
+  "network": "1",
+  "symbol": "DIA",
+  "domain": "vote.diadata.org",
```

You will need `skin` field to add your custom styles.
```suggestion
  "skin": "diadata",
  "domain": "vote.diadata.org",
```

---

To add your custom domain you will have to add your domain in `spaces/domains.json` file also.
https://docs.snapshot.page/guides/custom-domain

---

Added, thanks